kratos/python_scripts/vtk_output_process.py

```diff
@@ -15,6 +15,12 @@ def __init__(self, model, settings ):
         model_part_name = settings["model_part_name"].GetString()
         self.model_part = model[model_part_name]
 
+        if settings.Has("write_properties_id"):
```

Comment:
I would not add a deprecation check here, this was added very recently and is most probably not used by many yet
Kratos is already full of deprecation-warnings that usually stay for very long

Reply:
I am happy with that, I used a few times

Reply:
Can of course remove the warning, up to you

Reply:
I would prefer to keep it a bit clean
In many cases I am the one cleaning things like this up, otherwise we will suffocate from too much deprecation warnings :D

Last choice up to you @loumalouomega since you use it, won't block

Reply:
If it is only used in a few cases for debug purposes i would also go for removing it.

Reply:
@loumalouomega what do you say, do you wan't keep the deprecation warning or can you change the name of the setting in your cases?